templates/admin/group/group_property_editor.html.ep

```diff
@@ -0,0 +1,115 @@
+                </div>
+                <div class="form-group">
+                    <label for="editor-keep-results-in-days" class="col-sm-2 control-label" title="currently not used">Keep results for</label>
+                    <div class="col-sm-10">
+                        <input type="number" min="0" class="form-control" id="editor-keep-results-in-days" name="default_keep_results_in_days" value="<%= $group->default_keep_results_in_days %>"> days
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label for="editor-keep-important-results-in-days" class="col-sm-2 control-label" title="currently not used">Keep important results for</label>
+                    <div class="col-sm-10">
+                        <input type="number" min="0" class="form-control" id="editor-keep-important-results-in-days" name="default_keep_important_results_in_days" value="<%= $group->default_keep_important_results_in_days %>"> days
+                    </div>
+                </div>
+            % }
+            <div class="form-group">
+                <label for="editor-default-priority" class="col-sm-2 control-label" title="currently not used">Default priority for jobs</label>
+                <div class="col-sm-10">
+                    <input type="number" class="form-control" id="editor-default-priority" name="default_priority" value="<%= $group->default_priority %>">
+                </div>
+            </div>
+            <div class="form-group">
+                <label for="editor-description" class="col-sm-2 control-label" title="shown on top of the group overview">Description</label>
+                <div class="col-sm-10">
+                    <textarea class="form-control" id="editor-description" name="description" ><%= $group->description %></textarea>
+                </div>
+            </div>
+            <div class="form-group">
+                <label for="editor-info" class="col-sm-2 control-label"><span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span></label>
+                <div class="col-sm-10">
+                    <span id="editor-info">All time-related properties (measured in days) can be set to <em>0</em> to denote infinity.</span>
+                </div>
+            </div>
+            <div class="form-group">
+                <div class="col-sm-offset-2 col-sm-10">
+                    <p class="buttons">
+                        <button type="submit" class="btn btn-primary"><span class="glyphicon glyphicon-floppy-disk" aria-hidden="true"></span> Save</button>
+                        <button type="reset" class="btn btn-danger"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span> Reset</button>
+                    </p>
+                    <p class="properties-progress-indication">
+                        <i class="fa fa-cog fa-spin fa-3x fa-fw"></i>
+                        <span class="sr-only">Loading...</span>
```

Comment:
For consistency I also replaced other user-visible occurrences of `...` in the web UI. Since this change has nothing to do with D&D editor I created an extra commit.